ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -30,25 +30,24 @@ import           GHC.Records
 import           NoThunks.Class (NoThunks)
 import           Numeric.Natural (Natural)
 
+import           Cardano.Binary (FromCBOR, ToCBOR)
+
 import           Cardano.Ledger.Allegra (AllegraEra)
-import qualified Cardano.Ledger.Core as LC
+import           Cardano.Ledger.Allegra.Translation ()
+import qualified Cardano.Ledger.Core as Core
 import           Cardano.Ledger.Era (Crypto, SupportsSegWit (..))
+import qualified Cardano.Ledger.Era as SL (TranslateEra (..), TxInBlock)
```

Comment:
`Cardano.Ledger.Era` was qualified above as `Core`. Maybe we should keep the same consistent naming, but also it could be the case that this module specifically deals with stuff very specific to `Cardano.Ledger.Era`. In that case I would maybe qualify it as `Era`?

Reply:
Ah; I have commit on the next PR that does that. I'll move it to this one.

I went with `.Era` because generally what we're getting from there is the same conceptual thing as we get from `.Core` --- even the module header on `.Era` says it exports "core" stuff.

Reply:
> Maybe it would be worth modifying other files to make the qualification more uniform across our code.

That is for Issue #3105, which I' don't want to address yet. I needed to rename them in the modules in this PR to avoid import conflicts, so I did it where necessary. There are several other modules with out-of-date imports.

We'll need to do a broader review and pick a Pareto-optimal scheme -- there are unfortunately multiple dimensions that come to mind.

Reply:
Perfect! resolving this thread then.